Source/Scene/BillboardCollection.js

```diff
@@ -1218,6 +1237,9 @@ define([
                 if (this._shaderPixelOffsetScaleByDistance) {
                     vs.defines.push('EYE_DISTANCE_PIXEL_OFFSET');
                 }
+                if (defined(this._globe)) {
```

Comment:
Hmm.  I hate to say that we always do it if it is available since it would hurt performance, and is especially overkill for space apps, for example.

Perhaps we could have a flag to enable this (regardless of clamp to ground) that defaults to false.  I don't know that I would expose it until we are sure it is needed though.